### DISCUSSION TITLE
VcDrawings里面的getSelectedDrawingActionInstance方法怎么调用？

### DISCUSSION THREAD

**Discussion by LivWei** [zouyaoji/vue-cesium#430] [General]
请问下getSelectedDrawingActionInstance如何使用
![image](https://github.com/zouyaoji/vue-cesium/assets/30063079/a85b5d30-56de-412d-a9fa-9a5ebbb37a86)

**Comment by zouyaoji** [comment]
绑定ref，通过ref来引用哈。
